api-csharp/samples/start.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ -z "$TRINSIC_ACCESS_TOKEN" ]; then
```

Comment:
Typescript now does not adhere to this "standard" as the only one. I kind of like that you are doing it like this but perhaps it should just be the same for all of them?